src/app/components/CreditPoolHeader/components/AppButton/AppButton.style.ts

```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from 'utils/styles';
+
+export const useStyles = makeStyles(
+  () => ({
+    root: {
+      padding: 0,
+    },
+  }),
+  { name: 'appButton' },
```

Comment:
I am using dash case (app-button) for such names, let's follow single naming scheme to keep consistency